actions/aws/lambda-deploy/action.yml

```diff
@@ -42,6 +46,7 @@ runs:
         aws-region: ${{ inputs.AWS_REGION }}
 
     - name: Update the Lambda code (new Docker image)
+      id: update_function
```

Comment:
Two githubActions can have the same ID ? Here `update_function` for both "update the lambda code" and "update the lambda alias".

Reply:
https://docs.github.com/fr/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsid

Reply:
`update_function_name` ?

Reply:
nice catch